plugins/gatsby-source-spryker-shop/lib/node.js

```diff
@@ -26,34 +26,42 @@ function createNodeData(operations, type, id, entity) {
     }
 }
 
+const conversionObjectToArrayCollection = collection => {
+    const modules = Object.values(collection.core);
+    const collectionOfFiles = [];
+    for (let module of modules) {
+        collectionOfFiles.push(...Object.values(module).reduce(
+            (accumulator, current) => [...accumulator, ...current], []
+        ));
+    }
+    return collectionOfFiles;
+};
```

Comment:
In case if we will expand styleguide to project level suggest to use recursive reducer

const conversionObjectToArrayCollection = collection =>
    Object.values(collection).reduce((accumulator, item) => {
        const currentItem = !Array.isArray(item) ? conversionObjectToArrayCollection(item) : item;

        return [...accumulator, ...currentItem];
    }, []);

and change parameter while invoke it to - modules.core

Reply:
Fixed without recurse.